src/main/resources/messages/messages.properties

```diff
@@ -342,6 +367,12 @@ error.tm.target_language_id_is_null='--target-language-id' must be specified alo
 error.tm.source_language_id_is_null='--source-language-id' must be specified along with '--target-language-id'
 error.tm.no_language_id='--language' is required for creating new translation memory
 
+error.task.list.unsupported_status=Unsupported status: '%s'
+error.task_is_not_added=Task was not added: '%s'
+error.task.unsupported.type=Unsupported task type. Possible values: translate, proofread
+error.task.empty_type=Task type can not be empty or null
```

Comment:
It would be great to specify possible values here also. Like in previous message

Reply:
done